network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/advanced_table_block.html

```diff
@@ -0,0 +1,51 @@
+{% load wagtailcore_tags %}
+
+<div class="article-table-block table-responsive">
+  <table class="table">
+    {% if self.caption %}
+      <caption>{{ self.caption }}</caption>
+    {% endif %}
+
+    {# Loop through every row #}
+    {% for row in self.table %}
+      {# If the "header" is checked, the first row will be a <thead> with a blue background #}
+      {% if self.header and forloop.first %}
+        <thead>
+          {% for cell in row.value %}
+            <th
+              scope="col"
+              {% if cell.value.column_width %}colspan="{{ cell.value.column_width }}"{% endif %}
+              class="blue {% if cell.value.centered_text %}text-center{% endif %}"
+            >
+              {{ cell.value.content|richtext }}
+            </th>
+          {% endfor %}
+        </thead>
+      {% else %}
+        {# For every other row that's not a header... #}
+        <tr>
+          {% for cell in row.value %}
+            {# If the first cell is suposed to be blue; when "column" is checked #}
```

Comment:
Typo in "suposed" :) 

Could we change "blue" to be "highlighted" for the same reason above? Thanks.